syft/workers/base.py

```diff
@@ -388,7 +396,7 @@ def execute_command(self, message: tuple) -> "pointers.PointerTensor":
             # function (i.e., torch.nn.functional.relu). Thus,
             # we need to fetch this function and run it.
 
-            sy.torch.command_guard(command_name, "torch_modules")
+            sy.torch.command_guard(command_name, "torch_modules")  # TODO[jvmancuso]: generalize
```

Comment:
same as above

Reply:
This is done, will update.

Reply:
https://github.com/OpenMined/PySyft/issues/2530